etc/adminhtml/system.xml

```diff
@@ -44,31 +44,82 @@
                     <label>Enabled</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="country" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Can we name this region?  I think this will cause confusion when we do EU as EU will be a region and not broken out by country

Reply:
Done